### ISSUE TITLE
Import Integration Wizard: 'No file select' after file upload

### ISSUE THREAD

**Issue by kahboom** [syndesisio/syndesis#3371]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ x ] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem
This is a minor bug, but in the Integration Import wizard, if you drag and drop a valid zip file, it will tell you that the file was successfully imported or has failed, but the input field still reads 'No file chosen'. Do we want to hide that text or display something else? I recall having this conversation with UXD and there was mention of redundancy, because the success/failure message already displays the file name, but we still should do something about the 'No file chosen' text, whether we choose to display the file name or hide it.

The failure error could also use some work, as you just get back a `null`, but I suppose that's a separate issue.

## Expected behavior
No confusion as to whether or not the file was uploaded or not.

## Screenshot

**Successful upload**

<img width="1679" alt="screenshot 2018-08-13 12 18 41" src="https://user-images.githubusercontent.com/3844502/44044532-d51fe4cc-9ef3-11e8-80a1-49609ab75dbd.png">

**Failed upload**

<img width="1680" alt="screenshot 2018-08-13 12 23 04" src="https://user-images.githubusercontent.com/3844502/44044537-d8c368ce-9ef3-11e8-8f1a-e603c7d28597.png">

## Tasks involved / Steps to Reproduce

1. Navigate to Integrations in the vertical navigation/sidebar.
2. If you don't already have an exported Integration zip file, then get one by selecting 'Export' on the right-hand side dropdown of any existing integration.
3. Click on the 'Import' button at the top right-hand corner. You'll be redirected to the Import wizard.
4.  Try to upload a zip file (preferably a valid one). See above.

cc @elvisisking

**Comment by dongniwang** [comment]
cc: @michael-coker

**Comment by michael-coker** [comment]
Thanks @kahboom, I opened https://github.com/syndesisio/syndesis/issues/3402 which should address this.

**Comment by kahboom** [comment]
@michael-coker - Shall I close this one then?

**Comment by michael-coker** [comment]
@kahboom yeah I think that's fine unless you want to leave it open and ensure it's fixed via #3402 after that issue is done.
